app/main.py

```diff
@@ -1,8 +1,41 @@
 class Car:
-    # write your code here
-    pass
+
+    def __init__(self, comfort_class: int, clean_mark: int, brand: str) \
+            -> None:
```

Comment:
Please, don't use ` \` to brake lines. Use the same logic for braking lines as you did in CarWashStation init method